### ISSUE TITLE
spawn EACCES Error

### ISSUE THREAD

**Issue by jonniespratley** [atom/grunt-electron-installer#22]
When trying to run the task I keep getting the following error:

```
Running "create-windows-installer" task
Verifying property create-windows-installer.appDirectory exists in config...OK
Reading /Users/predixgo/Github/grunt-electron-installer/vendor/Update.exe...OK
Writing /Users/predixgo/GithubEnterprise/pg-atomshell-container/dist/pxgo-1-win/Update.exe...OK
Reading /Users/predixgo/GithubEnterprise/pg-atomshell-container/dist/pxgo-1-win/resources/app/package.json...OK
Parsing /Users/predixgo/GithubEnterprise/pg-atomshell-container/dist/pxgo-1-win/resources/app/package.json...OK
Reading /Users/predixgo/Github/grunt-electron-installer/template.nuspec...OK
Writing temp/pg-atomshell-container.nuspec...OK
Warning: spawn EACCES Use --force to continue.

Aborted due to warnings.
```

Any thoughts? I've tried fixing the permissions on pretty much every directory that would be touched by this task, and still no solution.

**Comment by paulcbetts** [comment]
Are you trying to create a Windows installer while running on OS X?

**Comment by nbussman** [comment]
Is there any posibility to create a windows installer on a mac. I can't get it to work in my VM. Thanks fr your reply =)

**Comment by paulcbetts** [comment]
@nbussman You cannot create a Windows installer on Mac at the moment, sorry

**Comment by sergiovilar** [comment]
You guys should advice this in the README, I was trying to create the installer on OSX too.

**Comment by masom** [comment]
> You guys should advice this in the README, I was trying to create the installer on OSX too.

:+1: 


```
Running "create-windows-installer" task
Verifying property create-windows-installer.appDirectory exists in config...OK
Reading ./node_modules/grunt-electron-installer/vendor/Update.exe...OK
Writing ./release/apps/MyApp-win32/Update.exe...OK
Reading ./node_modules/grunt-electron-installer/template.nuspec...OK
Writing /var/folders/71/ycbr_06947l8_n96rvls0qsr0000gn/T/squirrel-installer-115515-4378-6vs3qy/myapp.nuspec...OK
Warning: spawn EACCES Use --force to continue.
```

It would be nice if there was an error message saying this can't be executed under a non-windows OS.

**Comment by hardchor** [comment]
What do we need to do to get this running on Linux/OSX? Wine?

**Comment by sergiovilar** [comment]
@hardchor I tried, can't work on non windows OSes, you could try (Appveyor)[http://www.appveyor.com/].

**Comment by michaelzoidl** [comment]
@masom +1

**Comment by paulcbetts** [comment]
> What do we need to do to get this running on Linux/OSX? Wine?

It's probably possible to get it working with Mono and Wine, but that will definitely require work

**Comment by paulcbetts** [comment]
This is fixed in #80
